css/main.css

```diff
@@ -16,5 +16,5 @@ body {
   font-family: "Raleway";
   font-style: normal;
   font-weight: 100;
-  src: local("Raleway Thin"), local("Raleway-Thin"), url("Raleway-Thin.ttf") format("truetype");
+  src: local("fonts/Raleway Thin"), local("fonts/Raleway-Thin"), url("fonts/Raleway-Thin.ttf") format("truetype");
```

Comment:
Similarly, we should probably remove of these two local references.

Reply:
I think this is on purpose, font-face can be a pain with hyphens depending on browser/OS.